src/Mpi/MpiKDTreeDecomposition.cpp

```diff
@@ -402,8 +402,8 @@ void MpiKDTreeDecomposition<ndim, ParticleType >::LoadBalancing
       int icell = mpitree->g2c[inode];
 
       // Create bounding boxes containing particles in each sub-tree
-      for (k=0; k<ndim; k++) mpinode[inode].domain.boxmin[k] = mpitree->tree[icell].boxmin[k];
-      for (k=0; k<ndim; k++) mpinode[inode].domain.boxmax[k] = mpitree->tree[icell].boxmax[k];
+      for (k=0; k<ndim; k++) mpinode[inode].domain.min[k] = mpitree->tree[icell].boxmin[k];
+      for (k=0; k<ndim; k++) mpinode[inode].domain.max[k] = mpitree->tree[icell].boxmax[k];
```

Comment:
Yes I think it would be more consistent with the rest of the code. I think MpiTree uselessly duplicates KDTree, but this is clearly outside the scope of this pull request